core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -132,6 +115,18 @@ case class PSBT(
     }
   }
 
+  def sign(
```

Comment:
nit: add a scaladoc with `@param` tags so that this is a nice interface for new users